src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/common/Event.java

```diff
@@ -10,9 +10,7 @@
     DOC_SIGNATURE_FAILURE, // Signature verification failure while processing zip file
     DOC_UPLOADED,
     DOC_UPLOAD_FAILURE,
-    DOC_PROCESSED, // when blob is successfully processed after storing all docs in DM. TODO remove when DB is cleaned
     DOC_CONSUMED, // client service handled the documents
-    BLOB_DELETE_FAILURE, // when blob is not successfully deleted after processing is complete
```

Comment:
I think there are events of this type in DB. If we ever try to read them we'll get an error.

Reply:
They were deleted in #1230. should I check the database again?

Reply:
Worth checking the count of each type of event. It looks like we could delete quite a few of them.

Reply:
actually, those were removed in a migration, oops